lib/collection/src/collection.rs

```diff
@@ -840,6 +841,85 @@ impl Collection {
         }
     }
 
+    pub async fn core_search_batch(
```

Comment:
We still have `search_batch`.

Can we remove it's body, transform the request parameters, and call into this function to remove duplicate code?

Reply:
Done in https://github.com/qdrant/qdrant/pull/2708